examples/platform/silabs/wifi/icd/ApplicationSleepManager.cpp

```diff
@@ -152,7 +152,7 @@ bool ApplicationSleepManager::ProcessKeychainEdgeCase()
 
     for (auto it = mFabricTable->begin(); it != mFabricTable->end(); ++it)
     {
-        if ((it->GetVendorId() == chip::VendorId::Apple) &&
+        if ((it->GetVendorId() == chip::VendorId::Apple || it->GetVendorId() == chip::VendorId::Google) &&
```

Comment:
Does the function description need to be updated in ApplicationSleepManager.h ?

Reply:
Updated the function description